car.capnp

```diff
@@ -291,6 +291,7 @@ struct CarControl {
   active @7 :Bool;
 
   actuators @6 :Actuators;
+  actuatorsFinal @10 :Actuators;
```

Comment:
@adeebshihadeh Any suggestions for a better name?

Reply:
maybe actuatorsOutput? I'd also add comments to explain both